utils_dev/oomph/idempiere.setup

```diff
@@ -26,7 +26,7 @@
   <setupTask
       xsi:type="setup:VariableTask"
       name="idempiere.target.platform"
-      defaultValue="idempiere-12-local-mirror"
+      defaultValue="idempiere-13-local-mirror"
```

Comment:
defaultValue must equals to the target name in https://github.com/idempiere/idempiere/blob/master/org.idempiere.p2.targetplatform/org.idempiere.p2.targetplatform.mirror.target

Reply:
done